packages/gatsby/src/utils/node-manifest.ts

```diff
@@ -320,37 +295,43 @@ export async function processNodeManifest(
  * and then removes them from the store.
  * Manifest files are added via the public unstable_createNodeManifest action in sourceNodes
  */
-export async function processNodeManifests(): Promise<void> {
+export async function processNodeManifests(): Promise<Record<string, string>> {
+  const startTime = Date.now()
   const { nodeManifests } = store.getState()
 
   const totalManifests = nodeManifests.length
 
   if (totalManifests === 0) {
-    return
+    return {}
```

Comment:
We should still return undefined or maybe null here. Later if we want to check what's returned we should check if it exists before using it